go.mod

```diff
@@ -9,8 +9,9 @@ require (
 	github.com/AdhityaRamadhanus/fasthttpcors v0.0.0-20170121111917-d4c07198763a
 	github.com/PuerkitoBio/purell v1.1.1
 	github.com/agrea/ptr v0.0.0-20180711073057-77a518d99b7b
+	github.com/alibaba/sentinel-golang v1.0.2 // indirect
 	github.com/cenkalti/backoff/v4 v4.1.1
-	github.com/dapr/components-contrib v1.3.0-rc1.0.20210804022438-7496adf1102b
+	github.com/dapr/components-contrib v1.3.0-rc1.0.20210814020225-284d74488f00
```

Comment:
@artursouza @pkedy It looks like we didn't publish a release tag with the v1.3.0 release for components-contrib ... is that intended?

Reply:
It's not preferable but not the end of the world, as long as this is referencing whatever is the last stable version of contrib that came out with the `1.3` release.